main.py

```diff
@@ -21,7 +24,23 @@
 
 PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
 
+# SERVER SETUP
+def flask_init():
```

Comment:
Can we __init__ this inside the WebService class?

https://github.com/BensGitHub2022/MT5-Expert-Advisor/blob/ben/webservice_example/src/web_service/web_service.py

see the webservice.py example